app/code/core/Mage/Core/etc/system.xml

```diff
@@ -62,19 +62,19 @@
                     </fields>
                 </csrf>
                 <cache>
-                    <label>Advanced Magento cache settings</label>
-                    <sort_order>10000</sort_order>
+                    <label>Advanced Cache Settings</label>
+                    <sort_order>1000</sort_order>
                     <show_in_default>1</show_in_default>
                     <show_in_website>0</show_in_website>
                     <show_in_store>0</show_in_store>
                     <fields>
                         <flush_cron_expr>
-                            <label>Cron expression for regular cache flushing</label>
+                            <label>Cron Expression for Flushing Cache</label>
                             <sort_order>1</sort_order>
                             <show_in_default>1</show_in_default>
                             <show_in_website>0</show_in_website>
                             <show_in_store>0</show_in_store>
-                            <comment>Use 0 0 30 2 0 to never run cron, the recommended setting is 30 3 * * 6</comment>
+                            <comment>Valid format: minute hour day(month) month day(week). Recommended value is 30 3 * * 6. Never run value is 0 0 30 2 0.</comment>
```

Comment:
I did not see your comment because I would have answered so far. From the perspective of the English language I referred to two distinct values, one called "Recommended" and the other one "Never run".  We can just as well specify "Don't run value is ...". If you want another short form to define that value I am open to changes.

Reply:
I would have preferred the original `Use "0 0 30 2 0" to never run.` but since I'm not english mothertongue I'll leave it up to you